doc/wg/core/notes/core-notes-2022-11-18.md

```diff
@@ -0,0 +1,222 @@
+ * Phil: So the resolution here is that the formatting expectations are about
+   the Tock project license headers/copyright notices.
+ * Leon: That is nice because it means this document does not cover copyright
+   enforcement
+ * Johnathan: My next comments says that if another project's license header we
+   should be willing to accept theirs rather than nearly duplicating it at the
+   top of the file
+ * Amit + Phil: I basically agree with that
+ * Johnathan: I think it is fair for the document to acknowledge that in cases
+   when files are pulled in from other projects, the format of those notices
+   might deviate slightly.
+ * Amit: So we can modify license headers to make them compatible, but we do
+   not need to if it is close enough?
+ * Johnathan: No we can't modify
+ * Leon: How will we specify which files are "Tock project files" to avoid
+   adding our license headers to other project files
+ * Leon: For example, if we pulled in a git submodule. Would we have a mechanism
+   to make our tool aware that this might not follow our guidelines?
+ * Johnathan: Google's practice is to have a directory called `third\_party`
+   that covers all code that might be under different licenses. we could copy
+   that practice, and the tool could ignore it. We might be able to get away
+   with skipping submodules.
+ * Leon: If we have any such mechanism, does this comment still apply?
+ * Johnathan: Yes, because we might copy in other files directly into the main
+   source e.g. driver files or vendoring code into libraries, and in those
+   examples we want to preserve original headers but also have a copyright that
+   reflects the possibility for continued additions by Tock contributors.
+ * Amit: OK, our conclusion is that if they are different enough we would want
+   to modify we should create a new license, but the tool can be liberal about
+   accepting license that are close to our expectations to avoid needless
+   additional headers.
+ * Amit: What about files that do not support comments, e.g. markdown? The
+   current distinction of code vs. not code might be ambiguous.
+ * Amit: I proposed any file type that allows comments should have a license
+   header
+ * Hudson: How precise a category is that? Does a text file "allow" comments?
+ * Johnathan: We obviously should not require people to add metadata in PNGs --
+   are those comments?
+ * Phil: The statement says code artifacts
+ * Johnathan: That is vague
+ * Johnathan: I think license headers in all textual files that allow comments
+   should suffice
+ * Amit: Sounds good *adds that comment to the PR on his shared screen*
+ * Amit: I don't think that the license that applies to input that affects a
+   binary should carry the same license as documentation. Does it even make
+   sense to licenses for a README?
+ * Johnathan: Unless our documentation is under a different license things get
+   weird, and our documentation includes copied code.
```

Comment:
I'm not quite sure what I said here, but it wasn't this. I think the following was in line with what I said:

`We didn't specify that our documentation is under a different license, so it is under the same license. Using a different license for our documentation would be weird because we copy code between our documentation and implementation`.